default-input.js

```diff
@@ -50,7 +50,11 @@ function readDeps (test, excluded) { return function (cb) {
 }}
 
 var name = package.name || basename
-var spec = npa(name)
+try {
+  var spec = npa(name)
```

Comment:
Maybe leave a `var spec` before the `try {`? This might work but I’m not sure off the top of my head whether that’s just because this file is lacking `use strict` …

Reply:
It does work because of hoisting, but I think it will look better to have `var spec` outside of the `try ... catch`.